corehq/apps/app_manager/models.py

```diff
@@ -2705,6 +2705,13 @@ def new_form(self, name, lang, attachment=Ellipsis):
         form = self.get_form(-1)
         if attachment == Ellipsis:
             attachment = get_blank_form_xml(name)
+        elif attachment == "default_followup_form":
+            msg = "This is your follow up form. Delete this label and add questions for any follow up visits."
```

Comment:
translate this

Reply:
How does translating in the app code work?

Reply:
With `ugettext` or `ugettext_lazy` - check this out: http://commcarehq.readthedocs.io/en/latest/translations.html